Sources/Auth/Model/AuthLoggable.swift

```diff
@@ -56,23 +57,23 @@ extension AuthLoggable {
 	
 	var loggingMetadata: Logger.Metadata {
 		var metadata = [String: Logger.MetadataValue]()
-		
+
 		switch self {
 		case .initializeDeviceLoginNetworkError(let error),
 			 .finalizeLoginNetworkError(let error),
 			 .finalizeDeviceLoginNetworkError(let error),
 			 .getCredentialsUpgradeTokenNetworkError(let error):
-			metadata[Self.metadataErrorKey] = .string(error.localizedDescription)
+			metadata[Self.metadataErrorKey] = "\(error.localizedDescription)"
```

Comment:
I've updated this so it follows as recommended in docs, which is to use literals and in thia case of strings, string interpolation, instead of `.string`:

```
    /// A logging metadata value. `Logger.MetadataValue` is string, array, and dictionary literal convertible.
    ///
    /// `MetadataValue` provides convenient conformances to `ExpressibleByStringInterpolation`,
    /// `ExpressibleByStringLiteral`, `ExpressibleByArrayLiteral`, and `ExpressibleByDictionaryLiteral` which means
    /// that when constructing `MetadataValue`s you should default to using Swift's usual literals.
    ///
    /// Examples:
    ///  - prefer `logger.info("user logged in", metadata: ["user-id": "\(user.id)"])` over
    ///    `..., metadata: ["user-id": .string(user.id.description)])`
    ///  - prefer `logger.info("user selected colors", metadata: ["colors": ["\(user.topColor)", "\(user.secondColor)"]])`
    ///    over `..., metadata: ["colors": .array([.string("\(user.topColor)"), .string("\(user.secondColor)")])`
    ///  - prefer `logger.info("nested info", metadata: ["nested": ["fave-numbers": ["\(1)", "\(2)", "\(3)"], "foo": "bar"]])`
    ///    over `..., metadata: ["nested": .dictionary(["fave-numbers": ...])])`
    public enum MetadataValue {
```